tests/cephadm/test_cephadm_ansible_wrapper_set_osd_config.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from cli.cephadm.ansible import Ansible
+from cli.cephadm.cephadm import CephAdm
+from cli.cephadm.exceptions import CephadmOpsExecutionError
+from utility.log import Log
+
+log = Log(__name__)
+
+CEPHADM_ANSIBLE_PATH = "/usr/share/cephadm-ansible"
+
+
+class ConfigNotFoundError(Exception):
+    pass
+
+
+def put_cephadm_ansible_playbook(node, playbook):
```

Comment:
better to have comman place for this function

Reply:
Added to cephci/cli/cephadm/ansible.py